utility/get_ip_addrs.py

```diff
@@ -0,0 +1,57 @@
+""" Look at Ceph storage pods, return ip-addresses """
+import subprocess
+import shlex
+
+
+def get_ip_addrs_for_ceph_type(pod_type, namespace='openshift-storage'):
+    """
+    Get the ip address for all pods of a given type.
+    Args:
+        pod_type (string): pod type (mon, mgr, osd, all for everbody)
+        namespace (string): namespace in -n field of oc commands
+    Returns:
+        List of ip_addresses for all pods of the type specified
+    """
+    ret_list = []
+    look_for = f"rook-ceph-{pod_type}"
+    ipaddrs = get_ip_addrs(namespace)
+    for ceph_pod in ipaddrs:
+        if ceph_pod[0].startswith(look_for):
+            ret_list.append(ceph_pod[1])
+    return ret_list
+
+
+def get_ip_addrs(namespace='openshift-storage'):
+    """
+    Get the ip addresses of all pods in the namespaces.
+    Args:
+        namespace (string): namespace in -n field of oc commands
+    Returns:
+        List of two item lists of the form [pod_name, ip_address]
+    """
+    ret_value = []
+    oc_cmd = f"oc -n {namespace} get pods -o wide"
```

Comment:
Please use OCP class utilities for performing oc operations.

Reply:
I was a little confused.  I thought that we should use the oc cli interface.  I will change this to call get_api_client("OCRestClient") instead.

Reply:
@wusui 
Please use function from https://github.com/red-hat-storage/ocs-ci/blob/0bf9d2c5464ec1d0f4eaae1c24e070a18ddee220/ocs/ocp.py#L45. Usage of REST client may be deferred for future.